scripts/fetch-android-examples.sh

```
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)